spec/support/usps_ipp_helper.rb

```diff
@@ -367,6 +378,17 @@ def stub_request_proofing_results_with_invalid_response
     )
   end
 
+  def stub_request_proofing_results(status_code: 200, body: {})
```

Comment:
[non-blocking] I just wanted to mention, I did like the `response_body_overrides` parameter from above, it's not necessary to add it here of course, but I was just curious if it would make sense in this method as well.

Reply:
Noting that it might be a bit more than we want to do in this PR

Reply:
It might be helpful... I just felt like refactoring that file in this PR would be a lot.

Reply:
I agree that refactoring `usps_ipp_helper` should be done in another ticket/PR. I think keeping the scope narrow for in-flight work is better.

Reply:
I created a story for this [LG-14751](https://cm-jira.usa.gov/browse/LG-14751)